doc/nvim-tree-lua.txt

```diff
@@ -872,11 +872,12 @@ NvimTreeWindowPicker
 
 There are also links to normal bindings to style the tree itself.
 
-Normal
-EndOfBuffer
-CursorLine
-VertSplit
-CursorColumn
+NvimTreeNormal
+NvimTreeEndOfBuffer
+NvimTreeCursorLine
+NvimTreeVertSplit (deprecated, use NvimTreeWinSeparator)
+NvimTreeWinSeparator
+NvimTreeCursorColumn
```

Comment:
Not sure why there were no `NvimTree` prefixe. We can remove.

Reply:
I think it was because i've specified the original groups, but it doesn't matter too much, i actually think it's clearer this way